float64[] state_nominal
float64[] control_nominal
float64[] state_error
---
float64[] control